lib/rules/newline-after-var.js

```diff
@@ -146,7 +152,7 @@ module.exports = {
          */
         function checkForBlankLine(node) {
             const lastToken = getLastToken(node),
-                nextToken = sourceCode.getTokenAfter(node),
+                nextToken = lastToken === sourceCode.getLastToken(node) ? sourceCode.getTokenAfter(node) : sourceCode.getLastToken(node),
```

Comment:
Sorry, I don't think I understand why this is needed 😟 . Could you explain me a bit more?